build.sbt

```diff
@@ -154,6 +154,8 @@ lazy val transactionImpl = (project in file("transaction-impl"))
     transactionApi,
     itemApi,
     tools,
+    // TODO: Remove this dependency when completedStatement with varargs it's included on lagom core
```

Comment:
Oh I didn't realize that `CassandraReadSideUtils` hadn't been moved to `tools`. Otherwise I would have suggested doing that, too. That's OK, this is only temporary until Lagom 1.3.6 is released (before the end of the week).